File: libnetwork/src/main/java/com/gystry/libnetwork/ApiResponse.java
package com.gystry.libnetwork;

/**
 * @author gystry
 * 创建日期：2020/7/3 17
 * 邮箱：gystry@163.com
 * 描述：
 */
public class ApiResponse<T> {
    public boolean success;
    public int status;
    public String message;
    public T body;
}

File: libnetworkkt/src/main/java/com/gystry/libnetworkkt/ApiResponse.kt
package com.gystry.libnetworkkt

/**
 * @author gystry
 * 创建日期：2021/3/18 11
 * 邮箱：gystry@163.com
 * 描述：
 */
data class ApiResponse<T>(var success: Boolean=false,
                          var status: Int=0,
                          var message: String="",
                          var body: T? =null)